Get incident by query small fix (#35176)
demisto/content: Packs/Base/ReleaseNotes/1_34_21.md

#### Scripts

##### DBotFindSimilarIncidentsByIndicators

> -  Fixed an issue where the **populate_fields** argument didn't handle pipe characters (|).
> 
> ##### DBotFindSimilarIncidents
> 
> -  Fixed an issue where the **populate_fields** argument didn't handle pipe characters (|).
> 
> ##### GetIncidentsByQuery
> 
> - Fixed an issue where the **populate_fields** argument didn't handle pipe characters (|).

Reviewer: ```suggestion
-  Fixed an issue where the *populateFields* argument didn't accept a pipe (|) as a separator.

##### DBotFindSimilarIncidents

-  Fixed an issue where the *populateFields* argument didn't accept a pipe (|) as a separator.

##### GetIncidentsByQuery

- Fixed an issue where the *populateFields* argument didn't accept a pipe (|) as a separator.
```
